src/App.jsx

```diff
@@ -1,32 +1,69 @@
 import { useEffect, useState } from 'react';
-
 import './App.css';
 import { fetchBrews } from './services/breweries';
 import BrewCard from './components/BrewCard/BrewCard';
 import Controls from './components/Controls/Controls';
+import background from './assets/pattern.png';
+import Button from '@mui/material/Button';
 
 function App() {
   const [breweries, setBreweries] = useState([]);
   const [type, setType] = useState('all');
   const [loading, setLoading] = useState(true);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [zipcode, setZipcode] = useState('');
 
+  // data call
   useEffect(() => {
     const fetchData = async () => {
-      const data = await fetchBrews(type);
+      const data = await fetchBrews(type, currentPage, zipcode);
```

Comment:
I might look into a way to reset the page back to 1 when you select a different type or search by zipcode to avoid ending up with a blank page